run.sh

```diff
@@ -127,6 +127,13 @@ while true; do
     fi
   done
 
+  # ethtool for every interface
+  for iface in $(ip link | awk '/^[0-9]+: / {print $2}' | sed 's/:$//' | sed 's/@.*$//'); do
+    f=ethtool_${iface}
+    c=$(ethtool -S ${iface})
```

Comment:
is this ok running within container?

Reply:
with host network it should be fine

Reply:
I assuming dumping the host side of the veth is sufficient
We can poke into each NS for completeness